api/src/test/java/uk/gov/dhsc/htbhf/claimant/ClaimantLifecycleIntegrationTests.java

```diff
@@ -57,6 +57,7 @@
      * ........................................................| email about upcoming changes to payment
      */
     @Test
+    @SuppressWarnings("checkstyle:VariableDeclarationUsageDistance")
```

Comment:
The line `LocalDateTime now = LocalDateTime.now();` is considered too far above the first usage.